github-profiles/alex_aponte-4195/README.md

```diff
@@ -0,0 +1,26 @@
+### Hi there, I'm Alex 👋
+
+- ⚡ Electronic Engineer
+- Technology enthusiast
+- Linux user
+- 🌱 I’m currently learning JavaScript, Node.js, React
+
+*Funny terminal commands*
+
+Command    |Description   |
```

Comment:
It is visually recommended that the md table be aligned

Reply:
Resolved. Thanks